tests/golden.csv

```diff
@@ -1,988 +1,1437 @@
-Shashank Vineet,washingtontimes.com,2020-06-02,https://www.washingtontimes.com/news/2020/apr/21/news-kim-jong-un-heart-surgery-sparks-flurry-specu/,TRUE,en-us,2020-04-21 00:00:00,,News of Kim Jong-un heart surgery sparks flurry of speculation
-Shashank Vineet,washingtontimes.com,2020-06-02,https://www.washingtontimes.com/news/2020/apr/21/reports-kim-jong-un-health-not-true-south-korea-sa/,TRUE,en-us,2020-04-21 00:00:00,,"Reports of Kim Jong-un health 'not true,' South Korea says"
-Shashank Vineet,washingtontimes.com,2020-06-02,https://www.washingtontimes.com/news/2020/apr/21/robert-obrien-us-intelligence-monitoring-kim-jong-/,TRUE,en-us,2020-04-21 00:00:00,,Robert O'Brien: U.S. intelligence monitoring Kim Jong-un condition 'very closely'
-skip,worldnewsking.com,2020-06-02,https://worldnewsking.com/,FALSE,,,,
-skip,worldnewsking.com,2020-06-02,https://worldnewsking.com/2020/,FALSE,,,,
-skip,worldnewsking.com,2020-06-02,https://worldnewsking.com/2020/04/,FALSE,,,,
-skip,worldnewsking.com,2020-06-02,https://worldnewsking.com/2020/04/21/,FALSE,,,,
-skip,worldnewsking.com,2020-06-02,https://worldnewsking.com/2020/04/21/kim-jong-un-health-rumors-fueled-by-north-koreas-own-secrecy/,FALSE,,,,
-Mike Izbicki,wsj.com,2020-05-30,https://www.wsj.com/,FALSE,,,,
-Mike Izbicki,wsj.com,2020-05-30,https://www.wsj.com/articles/north-koreas-strong-hand-against-the-u-s-11572632466,TRUE,en-us,2019-11-01 18:21:00+00:00,2019-11-01 18:21:00+00:00,North Korea's Strong Hand Against the U.S.
-,wsj.com,2020-06-06,https://www.wsj.com/news/us,FALSE,,,,
-Shashank Vineet,yournews.com,2020-06-02,https://yournews.com/,FALSE,,,,
-Shashank Vineet,yournews.com,2020-06-02,https://yournews.com/2020/04/21/1578589/china-u-s-a-primitive-society-american-democracy-is-dying/,TRUE,en-us,2020-04-21 00:00:00,,"China: U.S. a 'Primitive Society,' 'American Democracy Is Dying'"
-Shashank Vineet,zimbabwevoice.com,2020-06-02,https://zimbabwevoice.com/,FALSE,,,,
-Shashank Vineet,zimbabwevoice.com,2020-06-02,https://zimbabwevoice.com/2020/,FALSE,,,,
-Shashank Vineet,zimbabwevoice.com,2020-06-02,https://zimbabwevoice.com/2020/04/,FALSE,,,,
-Shashank Vineet,zimbabwevoice.com,2020-06-02,https://zimbabwevoice.com/2020/04/21/,FALSE,,,,
-Shashank Vineet,zimbabwevoice.com,2020-06-02,https://zimbabwevoice.com/2020/04/21/north-korean-leader-kim-jong-un-critically-ill-after-surgery-reports/,TRUE,en-us,2020-04-21 04:48:05+00:00,2020-04-21 04:48:11+00:00,North Korean leader Kim Jong-Un 'critically ill' after surgery: Reports - Zimbabwe Voice
-Shashank Vineet,zimbabwevoice.com,2020-06-02,https://zimbabwevoice.com/2020/04/21/south-korea-denies-kim-jong-un-is-in-grave-danger-after-reported-heart-surgery/,TRUE,en-us,2020-04-21 07:30:37+00:00,2020-04-21 07:31:47+00:00,South Korea denies Kim Jong-un is in 'grave danger' after reported heart surgery
+human_annotator,hostname,download_date,url,is_article,lang,timestamp_published,timestamp_modified,title,
```

Comment:
You're changing every line of the `golden.csv` file again in this commit.  It looks like you've added a `,` at the end of every line.  This makes it difficult for me to tell which changes you've made are substantive and which are just cosmetic.